projects/rocprofiler-systems/source/lib/rocprof-sys/library/rocprofiler-sdk/counters.cpp

```diff
@@ -32,6 +34,40 @@ namespace rocprofiler_sdk
 {
 namespace
 {
+void
+metadata_initialize_counter_category()
+{
+    trace_cache::get_metadata_registry().add_string(
+        trait::name<category::rocm_counter_collection>::value);
+}
+
+void
+metadata_initialize_counter_track(const char* name)
+{
+    trace_cache::get_metadata_registry().add_track({ name, std::nullopt, "{}" });
+}
+
+void
+metadata_initialize_counters_pmc(size_t dev_id, const std::string& name,
+                                 const std::string& metric_description)
+{
+    size_t      EVENT_CODE       = 0;
+    size_t      INSTANCE_ID      = 0;
```

Comment:
Can these be `const`? They don't appear to be modified.

```suggestion
    const size_t EVENT_CODE       = 0;
    const size_t INSTANCE_ID      = 0;
```